javarun/__init__.py

```diff
@@ -0,0 +1 @@
+from .jrun import resolve_dependencies
```

Comment:
We could also import `jrun_main` here.
```suggestion
from .jrun import resolve_dependencies, jrun_main
```